### PR TITLE
Comms: New v11 release page 🎉  

### PR DESCRIPTION
**Pull request by janchild**
This PR adds a new page to support our v11 release communication efforts. It's in the What's happening section of our website and includes:

- an overview (bullet points that define the release in less than 60 seconds)
- links to the Medium articles
- extensive FAQs for designers and developers to set expectations of the experience of upgrading (or not).

### CODE REVIEW

**Comment by janchild** [comment]
@jeanservaas Would love to get your eyeballs on the FAQ section. I ended up adding some extra space between the bullet points because there’s just a lot of text and the extra space makes it easier to read. I’ve probably edited this stuff down by at least 1/3 already so it’s not going to get any shorter. Hopefully this is okay, or if you have any other ideas let me now. 

One thing to keep in mind with any other ideas... We wanted to keep the questions as H3s so we can direct link to them from other places. Thanks!

**Review by jillianhowarth — APPROVED** [review]
Just a small typo but this is awesome! 🎉

File: src/pages/whats-happening/v11-release/index.mdx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+- Inline theming for light/dark modes
+- Token renaming for better understanding and ease of use
+- Size props introduced for icons to reduce icon package size
+- Reduced compile times with new Sass modules
+
+### What's not changing
+
+- No changes to the IBM Design Language
+- No need to rebuild your UI
+- Tokens that are changing will continue to work in v1
```
Comment (mattrosno) [inline src/pages/whats-happening/v11-release/index.mdx:47]: >Tokens that are changing will continue to work in v11

File: src/pages/whats-happening/v11-release/index.mdx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+- We will continue to accept any contributions that look to address issues in
+  the v10 release if we are unable to get to them in time for your product
+  roadmap.
+
+### What is the rule for supporting deprecated things?
+
+- Things that were deprecated from v9 to v10 will be removed in v11.
+
+* Things that are deprecated in v10 will remain in v11 and will be removed in
+  v12.
```
Comment (mattrosno) [inline src/pages/whats-happening/v11-release/index.mdx:140]: I'm wondering if we should state our intent with timing of v12. We don't know when that would happen, and like v11 we want to delay breaking changes as long as it makes sense.

In the past, our rule was "major versions at minimum six months apart". But having gone over 2 years between v10 and v11, it seems we're set up for a slower frequency of major versions.

Maybe something like:

>The timeline of v12 has not been set, but it's the team's intent for major versions to be at minimum 12 months apart.

Thoughts @alisonjoseph @joshblack?

File: src/pages/whats-happening/v11-release/index.mdx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+  for more details.
+
+### How big is the expected effort to migrate code to v11?
+
+- The Carbon team is providing an upgrade experience to automate as much of the
+  migration process as possible.
+
+* A lot of the changes made will be name-based.
+
+- Some changes with components will require updating to the new Component API.
```
Comment (mattrosno) [inline src/pages/whats-happening/v11-release/index.mdx:192]: Lowercase "component API"?

File: src/pages/whats-happening/v11-release/index.mdx [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+* Teams will need to update their Sass dependency since `node-sass` has been
+  deprecated. They will also need to update how they bring in Carbon.
```
Comment (mattrosno) [inline src/pages/whats-happening/v11-release/index.mdx:195]: The latest medium article has good, detailed information on things teams can do now to prepare for the v11 release, like switching from `node-sass` to `sass` and updating the `bx` prefix. Maybe link to the article in this bullet point for more info on what can be done now?

https://medium.com/carbondesign/carbons-2021-release-april-update-9d23242b3dea

**Review by mattrosno — COMMENTED** [review]
Great words! Some questions and suggestions.

File: src/pages/whats-happening/v11-release/index.mdx (hunk 3 above)
Comment (janchild) [reply src/pages/whats-happening/v11-release/index.mdx]: Good catch. Thank you.

File: src/pages/whats-happening/v11-release/index.mdx (hunk 2 above)
Comment (janchild) [reply src/pages/whats-happening/v11-release/index.mdx]: Added. Let me know if tweaks needed.

File: src/pages/whats-happening/v11-release/index.mdx [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+- No new functionality will be introduced in v10 after the v11 release.
+
+* We will continue to address high impact bugs that come up for the v10 release
+  after the v11 release.
+
+- We will continue to accept any contributions that look to address issues in
+  the v10 release if we are unable to get to them in time for your product
+  roadmap.
+
+### What is the rule for supporting deprecated things?
```
[the remark this answers is not in the document]
Comment (janchild) [reply src/pages/whats-happening/v11-release/index.mdx]: good catch. thank you!

File: src/pages/whats-happening/v11-release/index.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (janchild) [reply src/pages/whats-happening/v11-release/index.mdx]: Oops! Thank you.

**Comment by aagonzales** [comment]
Here's an idea: should we update the tile in the hero banner on the home page to like to this v11 stuff? Its easy to overlook if you never open that "what's happening" tab.

File: src/pages/whats-happening/v11-release/index.mdx [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+- Token renaming for better understanding and ease of use
+- Size props introduced for icons to reduce icon package size
+- Reduced compile times with new Sass modules
+
+### What's not changing
+
+- No changes to the IBM Design Language
+- No need to rebuild your UI
+- Tokens that are changing will continue to work in v11
+- There's no rush to update if what you have today works for you
```
Comment (aagonzales) [inline src/pages/whats-happening/v11-release/index.mdx:48]: This line concerns me a little. It also implies they don't ever need to update.

File: src/pages/whats-happening/v11-release/index.mdx [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+
+## Read more in Medium
```
Comment (aagonzales) [inline src/pages/whats-happening/v11-release/index.mdx:50]: Are these the images Thy is working on? Are they supposed to have the same image for each article?

**Review by aagonzales — APPROVED** [review]
Once we figure out what we're doing with inline theming and the light variants we'll probably want to add a bullet or two for than but its looks good for now.

File: src/pages/whats-happening/v11-release/index.mdx (hunk 7 above)
Comment (janchild) [reply src/pages/whats-happening/v11-release/index.mdx:50]: These are the images that Jeannie and Thy have been working on. They build out in detail and color up through to the final article.

File: src/pages/whats-happening/v11-release/index.mdx (hunk 2 above)
Comment (alisonjoseph) [reply src/pages/whats-happening/v11-release/index.mdx]: Yes, I don't see us doing another major release anytime before next year, and even then, likely longer.

**Comment by janchild** [comment]
> Here's an idea: should we update the tile in the hero banner on the home page to like to this v11 stuff? Its easy to overlook if you never open that "what's happening" tab.

Yes, we have the banner in our comms plan. The banner would link to this page, so this is the first step.

File: src/pages/whats-happening/v11-release/index.mdx (hunk 6 above)
Comment (alisonjoseph) [reply src/pages/whats-happening/v11-release/index.mdx:48]: Could we re-word that last bullet point somehow "update at your convenience"  or just remove it?

Comment (janchild) [reply src/pages/whats-happening/v11-release/index.mdx:48]: I'm good with removing it if that works for @mattrosno and @joshblack

**Review by joshblack — APPROVED** [review]

File: src/pages/whats-happening/v11-release/index.mdx (hunk 6 above)
Comment (mattrosno) [reply src/pages/whats-happening/v11-release/index.mdx:48]: >There's no rush to update if you don't want to immediately take advantage of v11's features

Maybe? There was strong urgency to get on v10. The more we can set new expectations on how that's not the case with v11, the better.

Comment (alisonjoseph) [reply src/pages/whats-happening/v11-release/index.mdx:48]: I like that Matt 👍

**Comment by janchild** [comment]
Thanks for the feedback, everyone! Next round is cooking. I added a link from our Releases page too, @mattrosno.

File: src/pages/whats-happening/v11-release/index.mdx (hunk 7 above)
Comment (aagonzales) [reply src/pages/whats-happening/v11-release/index.mdx:51]: oh i see the checkboxes are colored in the second one, subtle.

File: src/pages/whats-happening/v11-release/index.mdx [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+- Token renaming for better understanding and ease of use
+- Size props introduced for icons to reduce icon package size
+- Reduced compile times with new Sass modules
+
+### What’s not changing
+
+- No changes to the IBM Design Language
+- No need to rebuild your UI
+- Tokens that are changing will continue to work in v11
+- There’s no rush to update if you don’t want to immediately take advantage of
```
Comment (alisonjoseph) [inline src/pages/whats-happening/v11-release/index.mdx:48]: Feedback from Phil -> Change from “no rush to update” to “when we will stop support for v10"

**Comment by alisonjoseph** [comment]
Feedback from @chrisconnors-ibm "What's not changing" should be at the top.

**Pull request merged**